test/e2e-go/cli/algoh/expect/algohTimeoutTest.exp

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/expect -f
+set err 0
+log_user 1
+
+
+
+if { [catch {
+    source algohExpectCommon.exp
+
+    set TEST_ALGO_DIR [lindex $argv 0]
+    set TEST_DATA_DIR [lindex $argv 1]
+
+    exec mkdir -p $TEST_ALGO_DIR
+    
+    set TIME_STAMP [clock seconds]
+    set timeout 5
+
+    set TEST_ROOT_DIR $TEST_ALGO_DIR/root
+    set TEST_PRIMARY_NODE_DIR $TEST_ROOT_DIR/Primary
+    set TEST_NODE_DIR $TEST_ROOT_DIR/Node
+    set NETWORK_NAME test_net_expect_$TIME_STAMP
+    set NETWORK_TEMPLATE "$TEST_DATA_DIR/nettemplates/TwoNodes50Each.json"
+    puts "TEST_ALGO_DIR: $TEST_ALGO_DIR"
+
+
+    ::Algoh::EstablishNetwork $NETWORK_NAME $NETWORK_TEMPLATE $TEST_ROOT_DIR
+
+    exec echo > $TEST_NODE_DIR/config.json \{ \"GossipFanout\": 1, \"EndpointAddress\": \"127.0.0.1:0\", \"DNSBootstrapID\": \"\", \"EnableProfiler\": false, \"IncomingConnectionsLimit\": 0, \"RunHosted\": true \}
+    exec echo > $TEST_NODE_DIR/host-config.json \{ \"DeadManTimeSec\": 32, \"StatusDelayMS\": 45000, \"StallDelayMS\": 60000, \"UploadOnError\": true, \"SendBlockStats\": false \}
+    
+    set PRIMARY_ADDR ""
+    spawn cat $TEST_ROOT_DIR/Primary/algod-listen.net
```

Comment:
there is a missing test for error code - you need to test the error code from this command.